apps/user-profile/src/components/modules/UserOnboardingModal/InterestBubble/InterestBubble.js

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from "react";
+
+import {
+  UserInterestCard,
+  UserInterestsCardContainer,
+  UserInterestsContainer,
+} from "./StyledInterestBubble";
+
+const UserInterests = () => {
```

Comment:
Believe it should be `const UserInterestBubble`.

This component should accept 'interests' as a parameter. And that 'interests' passed into the component should be whats used, instead of `const interests_list = ..`. 
So, `<UserInterestBubble interests={dataFromBackend} />`.